packages/radix-ui-themes/src/styles/utilities/grid-column.css

```diff
@@ -0,0 +1,63 @@
+@breakpoints {
+  .rt-r-gc {
+    /* Keep this selector here to maintain the source order */
+  }
+  .rt-r-gc-1 {
+    grid-column: 1;
+  }
+  .rt-r-gc-2 {
+    grid-column: 2;
+  }
+  .rt-r-gc-3 {
+    grid-column: 3;
+  }
+  .rt-r-gc-4 {
+    grid-column: 4;
+  }
+  .rt-r-gc-5 {
+    grid-column: 5;
+  }
+  .rt-r-gc-6 {
+    grid-column: 6;
+  }
+  .rt-r-gc-7 {
+    grid-column: 7;
+  }
+  .rt-r-gc-8 {
+    grid-column: 8;
+  }
+  .rt-r-gc-9 {
+    grid-column: 9;
+  }
```

Comment:
I'm probably not understanding all of this correctly, but if we support arbitrary values, why do we even need to generate all of this CSS?

Wouldn't it work anyway with via the `custom properties`?